tests/units/anta_tests/test_interfaces.py

```diff
@@ -25,6 +26,7 @@
 )
 from tests.lib.anta import test  # noqa: F401; pylint: disable=W0611
 
+# pylint: disable=C0302
```

Comment:
why? - please when you do this use the string version + lets make sure to try to not add this in the first place

Reply:
addressed in [4f9ba1d](https://github.com/arista-netdevops-community/anta/pull/552/commits/4f9ba1d79c52bb4b84a5225628f372cc6b8eaaf7)